admin/testing-changes-on-Docusaurus-itself.md

```diff
@@ -13,7 +13,9 @@ cd website
 npm run start
 ```
 
-If you look in the `website/package.json` file, you will notice that running `start` with `npm run` actually loads the local `start-server.js` file. This is how you know you are running with local code.
+## Necessary Files
```

Comment:
Probably don't need this header now, right? Can be removed? 

In fact, it might be appropriate to put the wording below in a note via the `>` markdown tag.